server/main.lua

```diff
@@ -14,14 +14,7 @@ local function isPlayerNearBus(src)
 end
 
 lib.callback.register('qb-busjob:server:spawnBus', function(source, model)
-    local netId = QBCore.Functions.CreateVehicle(source, model, Config.Location, true)
-    if not netId or netId == 0 then return end
-    local veh = NetworkGetEntityFromNetworkId(netId)
-    if not veh or veh == 0 then return end
-
-    local plate = locale('bus_plate') .. tostring(math.random(1000, 9999))
-    SetVehicleNumberPlateText(veh, plate)
-    TriggerClientEvent('vehiclekeys:client:SetOwner', source, plate)
+    local netId = SpawnVehicle(source, model, Config.Location, true)
```

Comment:
Return this immediately instead of creating a variable, also, why remove the custom plate?